client/src/styles/Card.css

```diff
@@ -1,95 +1,7 @@
-.card-parent-container {
-  height: 100vh;
-  max-height: 100vh;
-  padding-right: 56.25%;
+/* p {
```

Comment:
Do we need this file or is it a placeholder for future css?

Reply:
and similarly for other css files?

Reply:
I wasn't sure how others felt about doing the inline css versus breaking it out into separate css folders. Any preference? @jtenberg

Reply:
I think eventually we should separate it out if it becomes too cumbersome. If there isn't a lot of inline stuff, we can keep it the way it is. We can leave it for now but keep that as a consideration.